### PR TITLE
bump(main/cronie): v1.7.2

### PR DESCRIPTION
**Pull request by pranav10780**

### CODE REVIEW

**Comment by finagolfin** [comment]
You would also need to apply the version bump patch shown in #19705.

**Comment by pranav10780** [comment]
> You would also need to apply the version bump patch shown in #19705.

Sorry I don't understand what you meant

**Comment by finagolfin** [comment]
The linked issue tries to apply this version bump patch and fails:
```
diff --git a/packages/cronie/build.sh b/packages/cronie/build.sh
index 3b1f6c936..87431a253 100644
--- a/packages/cronie/build.sh
+++ b/packages/cronie/build.sh
@@ -3,9 +3,9 @@ TERMUX_PKG_DESCRIPTION="Daemon that runs specified programs at scheduled times a
TERMUX_PKG_LICENSE="ISC, BSD 2-Clause, BSD 3-Clause, GPL-2.0, LGPL-2.1"
TERMUX_PKG_LICENSE_FILE="COPYING, COPYING.obstack"
TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.7.1"
+TERMUX_PKG_VERSION="1.7.2"
TERMUX_PKG_SRCURL=https://github.com/cronie-crond/cronie/releases/download/cronie-${TERMUX_PKG_VERSION}/cronie-${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=78033100c24413f0c40f93e6138774d6a4f55bc31050567b90db45a2f9f1b954
+TERMUX_PKG_SHA256=f1da374a15ba7605cf378347f96bc8b678d3d7c0765269c8242cfe5b0789c571
TERMUX_PKG_AUTO_UPDATE=true
TERMUX_PKG_UPDATE_VERSION_REGEXP="\d+.\d+.\d+"
TERMUX_PKG_DEPENDS="dash"
```
Your patch does not bump the version like that, so you would need to do that too.

File: packages/cronie/fix-default-environment.patch [hunk 1]
```diff
@@ -21,18 +21,18 @@ diff -uNr cronie-1.5.5/src/env.c cronie-1.5.5.mod/src/env.c
...
++               "ANDROID_ROOT",
++               "ANDROID_RUNTIME_ROOT",
++               "ANDROID_TZDATA_ROOT",
++               "BOOTCLASSPATH",
++               "DEX2OATBOOTCLASSPATH",
++               "EXTERNAL_STORAGE",
++               "LD_PRELOAD",
++               "PREFIX",
++               "TMPDIR",
++               "TZ",
```
Comment (truboxl) [inline packages/cronie/fix-default-environment.patch:35]: Use tabs not spaces

**Comment by TomJo2000** [comment]
<sub>(This is a pre-written, saved reply.)</sub>
If you want to test this PR please download the appropriate DEB package(s)
from the build artifacts of the [associated PR's latest CI run](https://github.com/termux/termux-packages/pull/19834/checks).
image


After downloading the build artifact, make sure to `unzip` and un-`tar` it.
<details><summary>Detailed instructions, if needed.</summary>
<p>

```bash
# finding out what architecture you need
# architecture is just below the TERMUX_VERSION
termux-info

# e.g.
# [...]
# TERMUX_MAIN_PACKAGE_FORMAT=debian
# TERMUX_VERSION=0.118.0
# TERMUX__USER_ID=0
# Packages CPU architecture:
# aarch64
# [...]

# =======================

# make sure `unzip` and `tar` are installed using
pkg install unzip tar

# unzip the artifact (if you have a different architecture this might be arm, i686 or x86_64 instead)
unzip debs-aarch64-*.zip

# untar the artifact
tar xf debs-aarch64-*.tar

# You should now have a debs/ directory in your current working directory
# Install the packages from the local source using
pkg install -- ./debs/*.deb

# to clean up, you can remove the debs/ directory, .tar file and .zip file
rm -rfi debs debs-aarch64-*.zip debs-aarch64-*.tar
```

</p>
</details>

File: packages/cronie/fix-default-environment.patch (hunk 1 above)
Comment (TomJo2000) [reply packages/cronie/fix-default-environment.patch:35]: Your editor probably supports `.editorconfig`.
https://editorconfig.org/

We have one in the repository to automatically apply the right settings.

**Pull request closed without merging**